Thesis.tex

```diff
@@ -43,8 +43,8 @@
 \titleAP
 
 \begin{abstract}
-	\blindtext
-	% Pascals writes this
+	In our day and age, intellectual property is everywhere. One of the first pages of a book is often the copyright notice, one can not use a software without accepting its EULAs, and after the purchase of a film, one is notified what is considered appropriate use with a often unskippable notice. With a big legal apparatus hunting down offenders, one has to ask the question if the archaic laws of the past are still viable in our present age. In this paper we will provide a broad overview on what intellectual property even means, what notable organisations concern themselves with it, and how the current situation looks like. This analysis is centred on America by design, as it was a big contributor on the globalization, and did not just export its culture but also some of its laws. 
```

Comment:
purchase of a _movie_, one is ... with _an_ often unskippable notice.

In this paper _the authors_ will 

what intellectual property means (even weglassen), what has changed since the inception of intellectual property legislation, what notable...